recipes/lpsolve55/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "5.5" %}
+
+package:
+  name: lpsolve55
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/chandu-atina/lp_solve_python_3x.git
+  git_rev: 3d6ed38d8c3cfb63a281f2cd74ec23d97131351c
+  patches:
+    # https://github.com/chandu-atina/lp_solve_python_3x/pull/2
+    - sitedir.patch
+
+    # ftbfs on osx
+    - malloc.patch
```

Comment:
Do you want an `# [osx]` selector here?

Reply:
That header was only necessary for linux anyways,so we can leave it. The comment is enough.